lbrynet/core/Wallet.py

```diff
@@ -1321,6 +1321,23 @@ def get_address_balance(self, address, include_balance=False):
         else:
             return Decimal((float(c) + float(u) + float(x)) / COIN)
 
+    @defer.inlineCallbacks
+    def create_addresses_with_balance(self, num_addresses, amount, broadcast=True):
+        addresses = self.wallet.get_unused_addresses(account=None)
+        if len(addresses) > num_addresses:
+            addresses = addresses[:num_addresses]
+        elif len(addresses) < num_addresses:
+            for i in range(len(addresses), num_addresses):
+                address = self.wallet.create_new_address(account=None)
+                addresses.append(address)
+            yield self._save_wallet()
+
+        outputs = [[address, amount] for address in addresses]
+        # should this be defer_to_thread or defer_succeed ??
+        tx = yield self._run_cmd_as_defer_to_thread('paytomany', outputs)
```

Comment:
this should be run with _run_cmd_as_defer_succeed

    # run commands as a defer.succeed,
    # lbryum commands should be run this way , unless if the command
    # only makes a lbrum server query, use _run_cmd_as_defer_to_thread()